java/server/src/org/openqa/selenium/grid/server/BaseServer.java

```diff
@@ -106,7 +106,7 @@ public BaseServer(BaseServerOptions options) {
       FilterHolder
           filterHolder = servletContextHandler.addFilter(CrossOriginFilter.class, "/*", EnumSet
           .of(DispatcherType.REQUEST));
-      filterHolder.setInitParameter("allowedOrigins", "*");
```

Comment:
Because the default value of allowedOrigins is * (all origins), so it isn't necessary to set again at all.